doc/ref_arch/kubernetes/README.rst

```diff
@@ -59,11 +17,42 @@ Table of Contents
     chapters/chapter07
     chapters/appendix-a
 
-Required versions of most important components
+Required component versions
 ----------------------------------------------
 
 ========== ===================
 Component  Required version(s)
 ========== ===================
 Kubernetes 1.23
 ========== ===================
+
+Bundle/Version History
```

Comment:
remove as well?

Reply:
I think we should remove only the version history as I've added that to the top page of Anuket specs in #3077